components/form/form-element-mixin.js

```diff
@@ -76,12 +76,17 @@ export class FormElementValidityState {
 
 export const FormElementMixin = superclass => class extends LocalizeCoreElement(superclass) {
 
+	static get properties() {
+		return { validationError: { type: String } };
```

Comment:
I don't know if we need to explicitly prevent properties from reflecting, but I've seen `attribute: false` used here to at least prevent this getting set from an attribute.